giraffes.json

```
@@ -42,5 +42,6 @@
   "Julius Caesar was afraid of giraffes.",
   "Giraffes don't drink their bathwater.",
   "Giraffes do not yet have a seat at the United Nations.",
+  "Salvatore Dali wrote a screenplay for the Marx Brothers, _Giraffes on Horseback Salad_.",
```

Salvatore?

---

Looks like my autocorrect is Italian, not Spanish.  
```suggestion
  "Salvador Dali wrote a screenplay for the Marx Brothers, _Giraffes on Horseback Salad_.",
```